graphql/tests/query.rs

```diff
@@ -895,17 +895,18 @@ fn query_complexity_subscriptions() {
                 "subscription {
                 musicians(orderBy: id) {
                     name
-                    bands(first: 100, orderBy: id) {
+                    t1: bands(first: 100, orderBy: id) {
                         name
                         members(first: 100, orderBy: id) {
                             name
                         }
                     }
-                }
-                __schema {
-                    types {
-                        name
-                    }
+                    t2: bands(first: 200, orderBy: id) {
+                      name
+                      members(first: 100, orderBy: id) {
+                          name
+                      }
+                  }
```

Comment:
What happened here? Why does adding validations require changes to tests that check the complexity calculation?

Reply:
The subscription was failing now because of `SingleFieldSubscriptions` rule, since using root introspection fields along with a subscription is not valid. 
To achieve a similar complexity level, I changed the subscription to ask for more schema fields instead of introspection fields.

Reply:
Ah .. that makes sense. Thanks for clarifying!